.azure-pipelines/client.test.live.yml

```diff
@@ -7,6 +7,8 @@ jobs:
           pomFilePath: './batch/data-plane/pom.xml'
```

Comment:
Not sure if you meant to include this file in the PR. It doesn't seem related to checkstyle.

Reply:
The better name for this PR would probably be 'EventHubs integration part 1' I would guess.

Reply:
@mssfang Is it possible to split this PR into smaller ones? It'll make it easier to follow and CR. 190 files seems overwhelming, especially if some of them aren't to do with checkstyle.

Reply:
I think for future reference this is a good idea - smaller PRs more frequently works better than one big PR less frequently. However, I've reviewed this PR now and nothing stands out as troublesome to me. Let's not worry about closing and recreating smaller PRs for this one - let's just get this in once there is another +1.

Reply:
Sorry for the big PR created. Will create multiple smaller PR in the future.